### PR TITLE
GLT-2739: added Index Distance Tool tutorial

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: _includes/index-checking.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Click _Clear_ to reset the form and enter the combined sequences into the Index Distance
+Tool:
+
+```
+AAAAAACCCCCC
+AAAAAATTTTTT
+GAAAAACCCCCC
+```
+
+Click _Calculate_. Notice that the first two sequences are not considered indices even
```
Comment (avarsava) [inline _includes/index-checking.md]: "not considered indices"?

File: _includes/index-checking.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+<a name="index-distance-miso" href="#">top</a>
+
+# 5. Selecting Indices from MISO
+
+Rather than copying them from another source, or typing in the index sequences yourself,
+you can also choose indices from MISO to compare.
+
+1. Click _Clear_ to reset the form.
+1. From the _Indices_ list below the form, check the checkboxes beside any 5 indices
```
Comment (callunity) [inline _includes/index-checking.md]: Could change "check the checkboxes beside any 5 indices" to "select any 5 indices" for consistency with other documentation if you wanted.

File: _includes/index-checking.md (hunk 1 above)
Comment (djcooke) [reply _includes/index-checking.md]: duplicates. fixed

File: _includes/index-checking.md (hunk 2 above)
Comment (djcooke) [reply _includes/index-checking.md]: updated

**Pull request merged**